2022-04-29 18:58:28.284 [Info] Discovered 0 plugins:


--------------------------------------------------------------------------------
2022-04-29 18:58:29.457 [Info] Connecting to 38.129.153.213:5400...

--------------------------------------------------------------------------------
2022-04-29 18:58:29.884 [Info] Connection approved [00000000-0000-0000-0000-000000000000].

--------------------------------------------------------------------------------
2022-04-29 18:58:29.946 [Debug] approval Time=6575099 Offset=63786848934761 Real=63786855509861

--------------------------------------------------------------------------------
2022-04-29 18:58:29.946 [Debug] local Time=6575099 Offset=63786848934846 Real=63786855509946

--------------------------------------------------------------------------------
2022-04-29 18:58:29.946 [Debug] real delta=84

--------------------------------------------------------------------------------
2022-04-29 18:58:29.946 [Debug] this.Statistics.Ping=0 NCPing=5

--------------------------------------------------------------------------------
2022-04-29 18:58:29.948 [Debug] Connection established to remote [00000000-0000-0000-0000-000000000000/38.129.153.213:5400].

--------------------------------------------------------------------------------
2022-04-29 18:58:29.948 [Info] Connected [00000000-0000-0000-0000-000000000000].

--------------------------------------------------------------------------------
2022-04-29 18:58:42.971 [Warn] 'Me' is null.

--------------------------------------------------------------------------------
2022-04-29 18:58:43.554 [Debug] received epp: 6588161

--------------------------------------------------------------------------------
2022-04-29 18:58:46.670 [Debug] received epp: 6591879

--------------------------------------------------------------------------------
2022-04-29 18:58:47.084 [Error] Received unhandled exception from System.Threading.Tasks.ContinuationTaskFromTask.

--------------------------------------------------------------------------------
2022-04-29 18:58:47.084 [Error] AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread.

Caused By
ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
    Stack:    at System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
   at System.Collections.Generic.List`1.RemoveAt(Int32 index)
   at System.Collections.Generic.List`1.Remove(T item)
   at Intersect.Client.Framework.Content.ContentWatcher.<>c__DisplayClass12_0.<Modify>b__0(Task completedTask)
   at System.Threading.Tasks.Task.Execute()


--------------------------------------------------------------------------------
2022-04-29 18:58:47.086 [Error] Caused by:

--------------------------------------------------------------------------------
2022-04-29 18:58:47.086 [Error] ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
    Stack:    at System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
   at System.Collections.Generic.List`1.RemoveAt(Int32 index)
   at System.Collections.Generic.List`1.Remove(T item)
   at Intersect.Client.Framework.Content.ContentWatcher.<>c__DisplayClass12_0.<Modify>b__0(Task completedTask)
   at System.Threading.Tasks.Task.Execute()


--------------------------------------------------------------------------------
2022-04-29 18:58:47.088 [Info] Acquiring context lock... (0ms)

--------------------------------------------------------------------------------
2022-04-29 18:58:47.088 [Info] Acquired. (0ms)

--------------------------------------------------------------------------------
2022-04-29 18:58:47.089 [Info] Beginning context dispose. (0ms)

--------------------------------------------------------------------------------
2022-04-29 18:58:47.089 [Info] GC.SuppressFinalize (0ms)

--------------------------------------------------------------------------------
2022-04-29 18:58:47.089 [Info] InternalDispose() completed. (0ms)

--------------------------------------------------------------------------------
2022-04-29 18:58:47.089 [Info] Clearing instance... (1ms)

--------------------------------------------------------------------------------
2022-04-29 18:58:47.089 [Info] Releasing context lock... (1ms)

--------------------------------------------------------------------------------
2022-04-29 18:58:47.089 [Info] Released. (1ms)

--------------------------------------------------------------------------------
2022-04-29 18:59:31.587 [Error] Received unhandled exception from System.Threading.Tasks.ContinuationTaskFromTask.

--------------------------------------------------------------------------------
2022-04-29 18:59:31.587 [Error] AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread.

Caused By
ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
    Stack:    at System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
   at System.Collections.Generic.List`1.RemoveAt(Int32 index)
   at System.Collections.Generic.List`1.Remove(T item)
   at Intersect.Client.Framework.Content.ContentWatcher.<>c__DisplayClass12_0.<Modify>b__0(Task completedTask)
   at System.Threading.Tasks.Task.Execute()


--------------------------------------------------------------------------------
2022-04-29 18:59:31.587 [Error] Caused by:

--------------------------------------------------------------------------------
2022-04-29 18:59:31.587 [Error] ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
    Stack:    at System.ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument argument, ExceptionResource resource)
   at System.Collections.Generic.List`1.RemoveAt(Int32 index)
   at System.Collections.Generic.List`1.Remove(T item)
   at Intersect.Client.Framework.Content.ContentWatcher.<>c__DisplayClass12_0.<Modify>b__0(Task completedTask)
   at System.Threading.Tasks.Task.Execute()


--------------------------------------------------------------------------------
2022-04-29 19:00:01.636 [Debug] received epp: 6666833

--------------------------------------------------------------------------------
2022-04-29 19:00:02.442 [Debug] received epp: 6667662

--------------------------------------------------------------------------------
2022-04-29 19:00:02.998 [Debug] received epp: 6668224

--------------------------------------------------------------------------------
2022-04-29 19:00:03.775 [Debug] received epp: 6669004

--------------------------------------------------------------------------------
2022-04-29 19:00:05.386 [Debug] received epp: 6670599

--------------------------------------------------------------------------------
2022-04-29 19:00:16.955 [Debug] received epp: 6682178

--------------------------------------------------------------------------------
2022-04-29 19:00:17.193 [Debug] received epp: 6682395

--------------------------------------------------------------------------------
2022-04-29 19:00:17.393 [Debug] received epp: 6682598

--------------------------------------------------------------------------------
2022-04-29 19:00:17.601 [Debug] received epp: 6682818

--------------------------------------------------------------------------------
2022-04-29 19:00:17.839 [Debug] received epp: 6683051

--------------------------------------------------------------------------------
2022-04-29 19:00:18.268 [Debug] received epp: 6683489

--------------------------------------------------------------------------------
2022-04-29 19:00:27.052 [Debug] received epp: 6692271

--------------------------------------------------------------------------------
2022-04-29 19:00:27.304 [Debug] received epp: 6692520

--------------------------------------------------------------------------------
2022-04-29 19:00:28.053 [Debug] received epp: 6693254

--------------------------------------------------------------------------------
2022-04-29 19:00:29.469 [Debug] received epp: 6694692

--------------------------------------------------------------------------------
2022-04-29 19:00:29.526 [Debug] received epp: 6694692

--------------------------------------------------------------------------------
2022-04-29 19:00:30.643 [Debug] received epp: 6695865

--------------------------------------------------------------------------------
2022-04-29 19:01:02.401 [Debug] received epp: 6727613

--------------------------------------------------------------------------------
2022-04-29 19:01:02.601 [Debug] received epp: 6727630

--------------------------------------------------------------------------------
2022-04-29 19:01:03.814 [Debug] received epp: 6729020

--------------------------------------------------------------------------------
2022-04-29 19:01:04.032 [Debug] received epp: 6729254

--------------------------------------------------------------------------------
2022-04-29 19:01:04.309 [Debug] received epp: 6729521

--------------------------------------------------------------------------------
2022-04-29 19:01:04.990 [Debug] received epp: 6730207

--------------------------------------------------------------------------------
2022-04-29 19:01:05.483 [Debug] received epp: 6730709

--------------------------------------------------------------------------------
2022-04-29 19:02:12.970 [Warn] 'Me' is null.

--------------------------------------------------------------------------------
2022-04-29 19:02:13.102 [Debug] received epp: 6798207

--------------------------------------------------------------------------------
2022-04-29 19:05:00.445 [Error] InvalidOperationException: GetSkin: null
    Stack:    at Intersect.Client.Framework.Gwen.Control.Base.get_Skin()
   at Intersect.Client.Framework.Gwen.Control.Base.Dispose()
   at Intersect.Client.Interface.Debugging.DebugWindow.Dispose()
   at Intersect.Client.Interface.MutableInterface.DisposeDebugWindow()
   at Intersect.Client.Interface.Interface.DestroyGwen(Boolean exiting)
   at Intersect.Client.Core.Main.DestroyGame()
   at Intersect.Client.MonoGame.IntersectGame.Update(GameTime gameTime)
   at Microsoft.Xna.Framework.Game.DoUpdate(GameTime gameTime)
   at Microsoft.Xna.Framework.Game.Tick()
   at Microsoft.Xna.Framework.SdlGamePlatform.RunLoop()
   at Microsoft.Xna.Framework.Game.Run(GameRunBehavior runBehavior)
   at Intersect.Client.MonoGame.IntersectGame.MonoGameRunner.Start(IClientContext context, Action postStartupAction)
   at Intersect.Client.Core.ClientContext.InternalStart()
   at Intersect.Core.ApplicationContext`2.Start(Boolean lockUntilShutdown)


--------------------------------------------------------------------------------
